internal/datachannel/crypto.go

```diff
@@ -0,0 +1,362 @@
+package datachannel
+
+//
+// Code to perform encryption, decryption and key derivation.
+//
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/hmac"
+	"crypto/md5"
+	"crypto/sha1"
+	"crypto/sha256"
+	"crypto/sha512"
+	"errors"
+	"fmt"
+	"hash"
+	"log"
+
+	"github.com/ooni/minivpn/internal/bytesx"
+) //#nosec G501,G505
```

Comment:
Why are we disabling gosec here?

Reply:
both md5 and sha1 are in the weak crypto import blocklist:

``` 
G501: Blocklisted import crypto/md5: weak cryptographic primitive
G505: Blocklisted import crypto/sha1: weak cryptographic primitive
```

Reply:
Can you document it?